app/models/employee.rb

```diff
@@ -12,6 +12,6 @@ class Employee < ApplicationRecord
   has_many :rewards, through: :orders
 
   def number_of_points
-    received_kudos.count - rewards.sum(:price).to_i
+    received_kudos.count - orders.sum(&:order_snapshot_price)
```

Comment:
why not? then you can get rid of the `delegate` in order model :)
```suggestion
    received_kudos.count - orders.sum(&:order_snapshot.price)
```

Reply:
After the change I get the error `undefined method price`. After writing this method rubocop throws me the error and the proposal `delegate` and we are back to the same place again :)